### ISSUE TITLE
[Feature Request] Selectable sort order for chats

### ISSUE THREAD

**Issue by AlexanderAmelkin** [telegramdesktop/tdesktop#9846]
### Is your feature request related to a problem?

I have a lot of chats and sometimes some very active but non-important chats just preempt the inactive but important chats, even if the latter have unread messages.

### Describe the solution you'd like

It would be good if one could select the order in which the chats are sorted:

- By last activity
- By number of unread messages
- Alphabetically

I would also suggest to have an option to put unmuted chats first for all types of sorting.

### Describe alternatives you've considered

- Scrolling up and down to find the souce of the "unread messages present" indication
- Pinning the important chats as soon as a new message comes in and unpinning them or reading/replying

**Comment by shinenelson** [comment]
Not stale. Though #7461 is stale and can be closed. Its older, yes, but this one is better described properly according to the issue template.

**Comment by paymanz** [comment]
its really necessary feature because when your member of multiple large groups they jump up and down in the list and makes it really hard to select them.

**Comment by nklemiata** [comment]
Agree, "Scrolling up and down to find the source of the "unread messages present" indication" as mentioned is really annoying! Telegram is so rich in features, but does not have simple sorting..... 

It's been requested multiple times (here, in [17177](https://github.com/telegramdesktop/tdesktop/issues/17117), in [25261](https://github.com/telegramdesktop/tdesktop/issues/25261) and many others). 
I've recovered my Github login just to log it! :) 

Would be so great to have configurable sorting inside chat groups.

**Comment by f1yegor** [comment]
Still no progress in this direction?

**Comment by honglin-signo** [comment]
Yes, meanwhile can you add sort by last viewed? Sometimes I need to switch between group, for groups has no new msg, it will list in the bottom, hard to find

**Comment by 23rd** [comment]
Perhaps you will be a little bit happier with a new option `tg://settings/experimental/dialogs-unread-on-top`.

**Comment by Fran4end** [comment]
It would also be highly useful to have a setting for groups—which act as collections of topics—allowing you to arrange the topics in a fixed custom order.
